File: C++/1114.cpp
#include <iostream>
#include <string>
#include <limits>
#include <cmath>
#include <iomanip>

using namespace std;

int main()
{
    int x=0000;


    while (x != 2002)
    {
        cin >> x;
        x == 2002 ? (cout << "Acesso Permitido\n") : (cout << "Senha Invalida\n");
    }
}

File: JS/URI/1114.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let x = 0000;
let j=0;

while (x != 2002) {
    x = parseInt(lines[j].split(" ")[0]);
    console.log((x == 2002) ? "Acesso Permitido" : "Senha Invalida");
    j+=1;
    


}
